components/ArtGallery/Modal/index.js

```diff
@@ -5,46 +5,61 @@ export default function Modal({ images, onClose, activeImage, setActiveImage })
   // const handleClick = () => {
   //   onClose && onClose()
   // }
-  const handleControlTabClick = (e, url) => {
-    e.stopPropagation()
-    setActiveImage(url)
-  }
+  // const handleControlTabClick = (e, url, id) => {
+  //   e.stopPropagation()
+  //   setActiveImage({url, id})
+  // }
   // Special thanks to fireship.io for this beautiful code snippet to animate modal drop in.
```

Comment:
delete if no longer needed.

If possible it would be best to use the modal from headless UI here. You can see examples in the Answer Modal and Processing Modal. Their component should take care of the positioning on the screen and take care of closing when you click outside the modal.